Makefile.docker

```diff
@@ -67,7 +67,7 @@ else
 	@# 1. Copy appropriate coredns binary to build/docker/linux/<arch>
 	@# 2. Copy Dockerfile to build/docker/linux/<arch>
 	for arch in $(LINUX_ARCH); do \
-	    docker build -t $(DOCKER_IMAGE_NAME)-$${arch}:$(VERSION) build/docker/linux/$${arch} ;\
+	    docker build -t $(DOCKER_IMAGE_NAME)-$${arch}:$(VERSION) build/docker/$${arch}  && \
```

Comment:
I'm thinking  `image-download` should pull down images and put them into a directory structure that matches the release build paths, so that a user could build a custom coredns container (e.g. with external plugins), without needing to release the binaries to their GitHub fork as an intermediate step ... e.g. 

```
make -f Makefile.release build
make DOCKER=project -f Makefile.docker docker-build docker-push
```

This would also be useful for testing unreleased versions.